vendor/k8s.io/kubernetes/pkg/util/mount/exec_mount.go

```diff
@@ -112,12 +112,6 @@ func (m *execMounter) GetDeviceNameFromMount(mountPath, pluginDir string) (strin
 	return m.wrappedMounter.GetDeviceNameFromMount(mountPath, pluginDir)
 }
 
-// ResolveBindMountedBlockDevice resolves a block device bind mounted on mountPath by calling findmnt
```

Comment:
how did you update vendor packages? wouldn't Gopkg.lock get updated too?

Reply:
Actually, I directly updated them as a dirty hack, which should be fixed before merge.
As commented above, this dirty hack won't be necessary if 4aa6646 makes sense.